README.md

```diff
@@ -1,11 +1,72 @@
 [![Build Status](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client.svg?branch=master)](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client)
 
-pint
-====
+# Introduction
 
 pint (Public Cloud INformation Tracker) is a command line client to access
 the REST API provided by the SUSE Public Cloud Information Service. The
 information service provides data about the images SUSE publishes in various
 public cloud frameworks (Amazon, Google, Microsoft) as well as the
 servers that make up the SUSE maintained and operated update infrastructure
 in those frameworks.
+
+## Quick Start
```

Comment:
In general looks good. However, I would recommend either explicitly showing the use of virtualenvs to hold what is pip installed, or else modify the pip command lines to explicitly show the use of the `--user` option on the `pip install` commands to indicate that it should install to the `~/.local` rather than system infrastructure, and similarly the `sudo` command prefix to run the zypper command...